input/blog/2020-12-23-cake-vs-extension-release-0.3.0.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cake Visual Studio Extension Release 0.3.0
+category: Release Notes
+author: gep13
+---
+
+The Cake Team is very happy to announce the 0.3.0 release of the [Cake Extension for Visual Studio](https://marketplace.visualstudio.com/items?itemName=vs-publisher-1392591.CakeforVisualStudio).  This release has been baking for quite a while now, the previous release shipped in 2017, but we are hoping that it will be worth the wait.
```

Comment:
Link to VS page instead of directly the marketplace?

```suggestion
The Cake Team is very happy to announce the 0.3.0 release of the [Cake Extension for Visual Studio](/docs/integrations/editors/visualstudio/).  This release has been baking for quite a while now, the previous release shipped in 2017, but we are hoping that it will be worth the wait.
```

Reply:
I went with this, as this is what we had in the VSCode blog post, but I can get this fixed up.